<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addOwner</class>
 <widget class="QWidget" name="addOwner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>169</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>259</width>
    <height>169</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>259</width>
    <height>169</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sahip Ekle</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sahip ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Soyad</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="txtOwnerId"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txtOwnerName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txtOwnerLName"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>Ä°ptal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
